commands/build.go

```diff
@@ -891,17 +891,29 @@ func printResult(w io.Writer, f *controllerapi.PrintFunc, res map[string]string)
 	case "subrequests.describe":
 		return 0, printValue(w, subrequests.PrintDescribe, subrequests.SubrequestsDescribeDefinition.Version, f.Format, res)
 	case "lint":
-		err := printValue(w, lint.PrintLintViolations, lint.SubrequestLintDefinition.Version, f.Format, res)
-		if err != nil {
-			return 0, err
-		}
-
 		lintResults := lint.LintResults{}
 		if result, ok := res["result.json"]; ok {
 			if err := json.Unmarshal([]byte(result), &lintResults); err != nil {
 				return 0, err
 			}
 		}
+
+		if f.Format != "json" {
+			var warningCountMsg string
+			warningCount := len(lintResults.Warnings)
+			if warningCount == 1 {
+				warningCountMsg = "1 warning has been found!"
+			} else if warningCount > 1 {
+				warningCountMsg = fmt.Sprintf("%d warnings have been found!", warningCount)
+			}
+			fmt.Fprintf(w, "Check completed, %s\n", warningCountMsg)
```

Comment:
Can we have an integration test for both running check with and without warning to validate this?

iiuc this will print `Check complete, ` if no warnings are found and this as well https://github.com/docker/buildx/blob/07a85a544b4b1209377c2a74a9f5a91cb9d31d04/commands/?build.go#L924?